README.md

```diff
@@ -1 +1,49 @@
-# Portfolio-setup-and-mobile-first
\ No newline at end of file
+# Portfolio-setup-and-mobile-first
+
+> Description the project.
+
+- set up a new repository and prepare it for development using best practices (e.g. linters).
+- build the first 2 sections of the mobile website using the template you chose in the figma.
+
+## Built With
+
+- Major languages:
+
+* HTML and CSS
+
+- Technologies used:
+
+* Linters
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+Any one who wants to use this project should have at least the basic knowledge of HTML and CSS.
+
+### Install
+
+To install this project, copy the url of this project repo and simply run the following command to install.
+`git clone <url>`
+
+### Usage
+
+This project will help you to build a mobile first website of the front page of your portfolio website
+
+### Deployment
+
+This project is not deployed to any web hosting platfrom, but if any one wants to deploy this project feel free to do so.
+
+## Author
+
+- GitHub: [@KMT-Media](https://github.com/KMT-Media)
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- **[OPTIONAL]** Since you included the MIT license in your documentation, please consider adding the `MIT.md` file in your root directory as well. [Microverse MIT template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)
> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯

Reply:
Ok, thanks @Yazino12